### ISSUE TITLE
strange conversion

### ISSUE THREAD

**Issue by loicjaouen** [dhlab-basel/Sipi#230]
# original file

![original-crop](https://user-images.githubusercontent.com/3933654/39913358-dc79f762-5501-11e8-83aa-ca55a6c181b1.png)

# testing the conversion

- converting using sipi

```bash
docker run -ti -v $(pwd):/image dhlabbasel/sipi:latest /sipi/local/bin/sipi --format jpx -f /image/original-crop.png /image/sipi-crop-out.jpx
```

- result

<img width="365" alt="capture d ecran 2018-05-11 a 09 59 29" src="https://user-images.githubusercontent.com/3933654/39913417-0e181010-5502-11e8-87d6-8bb7c524e580.png">

- comparing using imagemagick

```bash
docker run -ti -v $(pwd):/image acleancoder/imagemagick-full:latest compare /image/original-crop.png /image/sipi-crop-out.jpx /image/compare-crop.png
```

![compare-crop](https://user-images.githubusercontent.com/3933654/39913462-3b93c232-5502-11e8-823a-cd423ccd4460.png)

**Comment by loicjaouen** [comment]
Dear Lukas (@lrosenth), many thanks for this quick feed back!

No, it is not the original file, it is a cropped version of a file that I converted from tiff to png.

On my laptop, I get the same result when converting the tiff and the png, when using the docker image, the tiff fails.

I will send you the original privately, but it can't be used for public use.

**Comment by loicjaouen** [comment]
Lukas will come with a solution to that.

In the mean time, following his advises, I tried to force the image to be RGB, skipped color profiles and forced a known color profile. None of that worked:

```bash
convert original.png -colorspace RGB /image/modified.png
convert original.png +profile "*" /image/modified.png
convert original.png -profile sRGB_v4_ICC_preference.icc /image/modified.png
```

After sipi conversion, the resulting _jpeg2000_ file still has wrong colors, and when opened in gimp it says that it has an embedded color profile and proposes convert to the default one that gimp is using.
So the profile does not hold in the png conversion.

As the original file is a tif, I pushed the [sRGB reference profile](http://www.color.org/srgbprofiles.xalter) to the tif file:
```bash
convert original.tif -profile sRGB_v4_ICC_preference.icc modified.tif
```

and sipi managed to convert it to a _jpeg2000_ nicely.
